frontend/integration-tests/views/login.view.ts

```diff
@@ -17,7 +18,7 @@ export const selectProvider = async(provider: string) => {
 };
 
 export const login = async(providerName: string, username: string, password: string) => {
-  if (providerName) {
+  if (await $('a.idp').isPresent()) {
```

Comment:
This will break our current login test. If the IDP that we create is still in flight when the login scenario runs, this condition will be false, meaning the IDP selection will be skipped, and we will attempt to login using test user credentials against the `kubeadmin` idp.